packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -324,23 +312,16 @@ const deleteUser = (id: string) => {
 const makeAdminToUserMutation = async (id: string | null) => {
   if (!id) {
     confirmAdminToUser.value = false;
-    toast.error(`${t('state.remove_admin_failure')}`);
+    toast.error(t('state.remove_admin_failure'));
     return;
   }
   const variables = { uid: id };
-  await adminToUser.executeMutation(variables).then((result) => {
-    if (result.error) {
-      toast.error(`${t('state.remove_admin_failure')}`);
-    } else {
-      toast.success(`${t('state.remove_admin_success')}`);
-      usersList.value = usersList.value.map((user) => {
-        if (user.uid === id) {
-          user.isAdmin = false;
-        }
-        return user;
-      });
```

Comment:
We'll need to preserve this change for the status change to reflect immediately (The admin badge goes away while performing).